.github/workflows/package-distribution.yaml

```diff
@@ -0,0 +1,63 @@
+name: Package distribution workflow
+on:
+  workflow_call:
+    inputs:
+      build-all:
+        description: "If the triggering workflow is running in build-all-components mode"
+        type: boolean
+        required: true
+      build-version:
+        description: "Specifies the build version to apply to all binaries produced by this workflow"
+        type: string
+        required: true
+      publish_binaries:
```

Comment:
nitpicky: some parameters use the dash as separator, others use underscore

Reply:
Thanks, fixed